scene/2d/particles_2d.cpp

```diff
@@ -235,7 +235,7 @@ _FORCE_INLINE_ static float _rand_from_seed(uint32_t *seed) {
 		s = 0x12345987;
 	k = s / 127773;
 	s = 16807 * (s - k * 127773) - 2836 * k;
-	if (s < 0)
+	if (s < 0)	// FIXME: but "s" is unsigned ?
```

Comment:
This is now fixed in master, so the addition of the comment should be rebased away as it conflicts.